jwst/extract_1d/tests/conftest.py

```diff
@@ -661,6 +663,52 @@ def mock_10_spec():
     model.close()
 
 
+@pytest.fixture()
+def mock_10_multi_int_spec():
+    """
+    Mock 10 simple spectra in a TSOMultiSpecModel.
+
+    Yields
+    ------
+    TSOMultiSpecModel
+        The mock model.
+    """
+    spec_list = []
+    for i in range(10):
+        spec_model = make_spec_model(name=f"slit{i + 1}", value=i + 1)
+        spec_list.append(spec_model)
+
+    tso_spec = ex.make_tso_specmodel(spec_list)
+    model = dm.TSOMultiSpecModel()
+    model.spec.append(tso_spec)
+
+    yield model
+    model.close()
+
+
+@pytest.fixture()
+def mock_2_multi_int_spec():
```

Comment:
does this fixture differ at all from `mock_10_multi` aside from the fact it makes a shorter list of spectra?  I'd suggest moving all the code to a function with an n_spectra parameter, then just making the fixtures each call that function with their respective number of spectra. This would avoid repeating code.